action.yml

```diff
@@ -40,7 +44,8 @@ runs:
       run: |
         set -ex
         for world in ${{ inputs.worlds }}; do
-          wit-bindgen markdown ${{ inputs.directory }} -w $world --check --html-in-md
+          ${parameter:+word}
+          wit-bindgen markdown ${{ inputs.directory }} -w $world --features ${{ inputs.features }}, --check --html-in-md
```

Comment:
Can the list of features be empty? So like `--features=""` won't fail but resolves to no features?

Reply:
The trailing comma in `${{ inputs.features }},` means an empty list will result in passing `--features ,`, which wit-bindgen interprets as an empty list.

Reply:
I guess `--features=${{ inputs.features }}` would be a clearer way to achieve that. I've now changed it to do that.